mne/coreg.py

```diff
@@ -77,9 +77,10 @@ def create_default_subject(mne_root=None, fs_home=None, update=False,
 
     Parameters
     ----------
-    mne_root : None | str
-        The mne root directory (only needed if MNE_ROOT is not specified as
-        environment variable).
+    mne_root :
```

Comment:
My concern was to also issue a deprecation warning for `create_default_subject(None, 'fs/path')` calls... Do you think we can ignore that case?